util/constants.js

```diff
@@ -0,0 +1,168 @@
+const basePath = 'http://localhost:5000';
+const apiAuthenticationName = 'Bearer';
+
+const userRole = 'User';
+
+const adminRole = 'Admin';
+
+const adminUser = {
+    'email': 'admin@admin.com',
+    'password': 'admin'
+};
+const nonAdminUser = {
+    'email': 'gosho@abv.bg',
+    'password': 'gosho'
+};
+
+const part = {
+    'title': 'Admin part title',
+    'model': 'Admin part model',
+    'year': 2000,
+    'description': 'Admin part description',
+    'price': 10,
+    'image': 'Admin part image'
+};
+
+const newPart = {
+    'model': 'Admin part model',
+    'title': 'Admin part title',
+    'description': 'Admin part description',
+    'year': 2000,
+    'image': 'Admin part image',
+    'price': 10,
+};
+
+const project = {
+    'title': 'test Project',
+    'model': 'test Model',
+    'year': 2018,
+    'description': 'This is test description',
+    'image': 'string',
+    'comments': [
+        'Test comment'
+    ]
+};
+
+const adminProject = {
+    'title': 'Admin Project',
+    'model': 'Admin Model',
+    'year': 2010,
+    'description': 'This is Admin test description',
+    'image': 'string',
+    'comments': [
+        'Admin test comment'
+    ]
+};
+
+const newAdminProject = {
+    'title': 'Admin Edit Project',
+    'model': 'Admin Edit Model',
+    'year': 2011,
+    'description': 'This is Admin test description',
+    'image': 'string',
+    'comments': [
+        'Admin edited comment'
+    ]
+};
+
+const successSignedUp = 'You have successfully signed up! Now you should be able to log in.';
```

Comment:
It is ok to keep the messages as constants for this projet.
Project which require multilanguage support will need a different apporach for messages. Ussualy we put them in some sort of key-value files. We use the keys in the code and the key is substitutedwith value from a file that coressponds to the languge selected by the user